### ISSUE TITLE
Add the rule `attr-value-omittion`

### ISSUE THREAD

**Issue by YusukeHirao** [markuplint/markuplint#279]
Incorrect:

```html
<img src="..." alt />
<input type="checkbox" checked="checked" />
```

Correct:

```html
<img src="..." alt="" />
<input type="checkbox" checked />
```

**Comment by YusukeHirao** [comment]
- #290
